pkg/internalapis/cnsoperator/config/cnsfilevolumeclient_crd.yaml

```diff
@@ -1,6 +1,6 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
```

Comment:
Can you also modify this version - `controller-gen.kubebuilder.io/version: v0.2.5` to `v0.6.2`

Reply:
Done.